python/src/server.py

```diff
@@ -61,5 +62,18 @@ def predict(self, req: MyPredictionRequest) -> MyPredictionResponse:
         # Optionally postprocess the prediction here
         return prediction
 
+    def evaluate(self, req: KonanServiceBaseEvaluateRequest) -> KonanServiceBaseEvaluateResponse:
+        """Evaluates the model based on passed predictions and their ground truths
+
+        Args:
+            req (KonanServiceBaseEvaluateRequest): includes passed predictions and their ground truths
+
+        Returns:
+            KonanServiceEvaluateResponse: the evaluation(s) of the model based on some metrics
```

Comment:
LGTM but might need to update the doc string name too.

Should we also showcase how to override feedback and evaluate classes too? or leave that to the docs?

Reply:
Will leave that for a later date since they are not strictly required